### PR TITLE
[docs]: installation and dependencies

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -1,27 +1,67 @@
...
+Note that we are exposing hosts `/var/run/docker.sock` to `abstruse` image. This is the only way you persist native `docker` performance.
+All builds or jobs will actually run on the host.
+
+You could also not expose `/var/run/docker.sock` to `abstruse` but performance will not be as it should be. At least we couldn't find appropriate
+storage-driver or storage-driver combination that this will run with good performance. This will actually run docker-in-docker and please try to avoid
+this while using `abstruse` docker image.
+
+3. Open Application
+
+Run your favourite `Chrome` browser and navigate to `http://localhost:6500`. You should see `abstruse` setup page which will guide you
```
Comment (Izak88) [inline docs/INSTALLATION.md]: favorite*

File: docs/DEPENDENCIES.md [hunk 2]
```diff
@@ -1,3 +1,107 @@
...
+* [Ubuntu Linux](#ubuntu-linux)
+* [MacOS](#macos)
+* [Windows](#windows)
+
+
+### Ubuntu Linux
+
+1. Install Node.JS
+
+There are many ways installing Node.JS on Ubuntu. We will use approach where you can later update, uninstall or use another version of Node.JS easily.
```
Comment (Izak88) [inline docs/DEPENDENCIES.md]: I think that 'We will use an approach' or 'the approach' would be better here...

File: docs/INSTALLATION.md (hunk 1 above)
Comment (jkuri) [reply docs/INSTALLATION.md]: 👍  thanks

File: docs/DEPENDENCIES.md (hunk 2 above)
Comment (jkuri) [reply docs/DEPENDENCIES.md]: updated.

**Pull request merged**